### PR TITLE
OJ-1369 manually bootstrap api with o auth into dev account

### PR DESCRIPTION
**Pull request by scronk1**
First pass of oauth and APIs

### CODE REVIEW

File: infrastructure/private-api.yaml
Comment (jkunle) [inline infrastructure/private-api.yaml]: I think we can remove these

**Review by jkunle — APPROVED** [review]

**Review by merlinc — CHANGES_REQUESTED** [review]
Sorry, I'm gonna be a jerk and block this until my bootstrap PR is merged as it's likely to cause conflicts with what you've got here, and I think it will be useful to demo how to get around them

(apart from that it looks good to me! :smile:  )

**Comment by scronk1** [comment]
> Sorry, I'm gonna be a jerk and block this until my bootstrap PR is merged as it's likely to cause conflicts with what you've got here, and I think it will be useful to demo how to get around them
> 
> (apart from that it looks good to me! 😄 )

No dramas 😄

**Comment by scronk1** [comment]
Covered by different PR

**Pull request closed without merging**
